cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -1,12 +1,13 @@
-from django.test import TestCase
+from django.test import TestCase, override_settings
 
 from wagtail.tests.testapp.models import SimplePage
 from wagtail.wagtailcore.models import Site
 
 import mock
 
 from v1.wagtail_hooks import (
-    RelativePageLinkHandler, check_permissions, form_module_handlers
+    RelativePageLinkHandler, check_permissions, form_module_handlers,
+    serve_latest_draft_page
```

Comment:
Is this import now unused because of the test rewrite?

Reply:
Interesting, neither isort nor flake8 complain about it.